### PR TITLE
LOG-1819: Move e2e crio_partial_log_test to functional

### PR DESCRIPTION
**Pull request by jcantrill**
### Description
This PR
* Moves the crio_partial e2e test to the functional framework
* Removes the crio_partial e2e test

/assign @vparfonov 

### Links
* https://issues.redhat.com/browse/LOG-1819

### CODE REVIEW

File: test/functional/framework.go [hunk 1]
```diff
@@ -462,7 +462,7 @@ func (f *FluentdFunctionalFramework) WritesNApplicationLogsOfSize(numOfLogs, siz
 func (f *FluentdFunctionalFramework) WriteMessagesToLog(msg string, numOfLogs int, filename string) error {
 	logPath := filepath.Dir(filename)
 	encoded := base64.StdEncoding.EncodeToString([]byte(msg))
-	cmd := fmt.Sprintf("mkdir -p %s;for n in {1..%d};do echo \"$(echo %s|base64 -d)\" >> %s;sleep 1s;done", logPath, numOfLogs, encoded, filename)
+	cmd := fmt.Sprintf("mkdir -p %s;for n in {1..%d};do echo \"$(echo %s|base64 -d)\" >> %s;done", logPath, numOfLogs, encoded, filename)
```
Comment (syedriko) [inline test/functional/framework.go:465]: @jcantrill This sleep is a bandaid for a flake where fluent doesn't "notice" this log.

Comment (syedriko) [reply test/functional/framework.go:465]: Did removal of this sleep fix that test?

Comment (jcantrill) [reply test/functional/framework.go]: reverted

**Pull request merged**